### DISCUSSION TITLE
Please share your builds or experiences of the project so far

### DISCUSSION THREAD

**Discussion by wezhunter** [wezhunter/ESP32_LinuxCNC_MotionController_RealTime#20] [Show and tell]
Would love to hear from anyone who is testing, developing or using the project

**Comment by rambros3d** [comment]
Hello wez, I bumped into your project while researching about LinuxCNC.
Your stuff is impressive, these things caught my attention:
- **ESP32** (these chips are really good! )
- **USB networking** (essentially you broke the norm, no one seems to use USB for LinuxCNC)
- high **step rates**

We will also be working on a CNC project from next month. Specifically a CNC control board and a Servo Driver using SimpleFOC. 
When you are back from your break, you can count me in. I can help with hardware and testing.

I'm really curious to explore what this project can do!

**Comment by wezhunter** [comment]
Hi @rambrosteam 
Thanks for the kind words and support. I'll let you know as soon as working on it again, hopefully soon! It'll be great to have testers.

I'm interested to hear about your project too - are you able to share any more info?

Cheers
